client/src/utils/checkoutFormModels/CheckoutFormvalidationSchema.js

```diff
@@ -0,0 +1,46 @@
+import * as yup from 'yup';
+import checkoutFormModel from './checkoutFormModel';
+
+const {
+  firstName,
+  lastName,
+  email,
+  phone,
+  addressOne,
+  city,
+  zipcode,
+  country,
+  cardType,
+  nameOnCard,
+  cardNumber,
+  expiryDate,
+  cvv,
+} = checkoutFormModel;
+
+// const visaRegEx = /^(?:4[0-9]{12}(?:[0-9]{3})?)$/;
```

Comment:
https://stackoverflow.com/questions/12011792/regular-expression-matching-a-3-or-4-digit-cvv-of-a-credit-card

Reply:
Yup matcher changed to string,  .matches regexp added for CVV